MetasysServices/Interfaces/IMetasysClient.cs

```diff
@@ -265,6 +265,16 @@ public interface IMetasysClient :IBasicService
         /// <inheritdoc cref="IMetasysClient.GetNetworkDevices(string)"/>
         Task<IEnumerable<MetasysObject>> GetNetworkDevicesAsync(string type = null);
 
+        /// <summary>
+        /// Gets all network devices by classification.
+        /// </summary>
+        /// <param name="classification">Optional classification as a string</param>
+        /// <exception cref="MetasysHttpException"></exception>
+        /// <exception cref="MetasysHttpParsingException"></exception>
+        IEnumerable<MetasysObject> GetNetworkDevicesByClassification(string classification = null);

```

Comment:
Actually I would deprecate the methods related to the Network Devices directly defined in the 'MetasysClient'.
(I would add it to the 'NetworkDevices' service)